src/common/constants.ts

```diff
@@ -142,6 +142,16 @@ export class Constants {
     public static amlWorkspaceDesc = "Azure Machine Learning Workspace";
     public static asaJobDesc = "Azure Stream Analytics Job";
     public static amlApiVersion: string = "2018-03-01-preview";
+
+    public static needSimulatorInstalledMsg = "You must have the 'iotedgehubdev' tool installed for Edge Hub simulation.";
+    public static updateSimulatorMsg = "Update your 'iotedgehubdev' tool to the latest for the best experience.";
+    public static failedInstallSimulator = "Failed to install 'iotedgehubdev' tool because of error:";
+    public static outputNoSimulatorMsg = "Cannot execute command since there is no 'iotedgehubdev' installed. Please install it first.";
+    public static pipNotFoundMsg = "'pip' not found. Cannot install 'iotedgehubdev' tool and will have to be installed manually.";

```

Comment:
` will have to be installed manually` -> `please install it manually` to clarify that it is the user himself/herself who needs to perform that action.